makefile

```diff
@@ -561,7 +560,17 @@ $(LIBFBPIC_C): $(libfbpicobjdir)/%.o: %.c $(LIBFB_H) | $(libfbpicobjdir)
 	$(QUIET_CC)$(CC) -fPIC $(ALLCFLAGS) -c $< -o $@
 
 $(libdir)/libfbmt.a: $(LIBFBMT_C) $(LIBFBMT_S) | $(libdir)
+ifeq ($(TARGET_OS),dos)
+  # Avoid hitting the command line length limit (the libfb.a ar command line
+  # is very long...)
+	$(QUIET)rm -f $@
+	$(AR) x $(DJDIR)/lib/libpthread.a
```

Comment:
Is it necessary to include libpthread and libsocket into libfbmt? I expected additional dependencies to be added to the default libs in fbc.bas:hAddDefaultLibs() (i.e. -lpthread -lsocket would be added to fbc's link command).

Reply:
I followed the building procedure used by Monochromator. Perhaps your solution could work as well, I haven't tested it

Reply:
In hAddDefaultLibs(), it's possible to check for `fbGetOption( FB_COMPOPT_MULTITHREADED )`, and only add the pthread/socket libs in that case. That would prevent them from being linked automatically in the non-mt case.